shared/product-make.include

```diff
@@ -93,8 +91,11 @@ $(OUT)/ocil-unlinked.xml: $(OUT)/xccdf-unlinked-resolved.xml $(SHARED)/$(TRANS)/
 $(OUT)/xccdf-unlinked-ocilrefs.xml: $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/ocil-unlinked.xml $(SHARED)/$(TRANS)/xccdf-ocilcheck2ref.xslt $(SHARED)/$(TRANS)/shared_constants.xslt
 	xsltproc -stringparam product "$(PROD)" -o $@ $(SHARED)/$(TRANS)/xccdf-ocilcheck2ref.xslt $<
 
-$(PRODUCT_REMEDIATIONS_OUTPUT):
-	$(SHARED)/utils/generate-from-templates.py --oval_version $(OVAL_VERSION) --input ./templates --output ./templates/output build
```

Comment:
Yes, I think so. I have added something like that in Debian/8/Makefile, have a look here: https://github.com/OpenSCAP/scap-security-guide/pull/1530/files#diff-e34a4cf03b4b9fbbca72e75cc917951eR20